database/connector.py

```diff
@@ -169,13 +189,16 @@ def insert_honeypot_events_credential_from_module_processor(ip, username, passwo
             "date": date,
             "username": username,
             "password": password,
-            "country": byte_to_str(IP2Location.get_country_short(byte_to_str(ip))),
-            "machine_name": network_configuration()["real_machine_identifier_name"]
+            "country": byte_to_str(IP2Location
+                                .get_country_short(byte_to_str(ip))),
+            "machine_name": network_configuration()
+                                    ["real_machine_identifier_name"]
```

Comment:
here also readability issue! not able to understand why real_machine_identifier_name is hanging on the next line

Reply:
older commit